<?xml version="1.0"?>
<launch>
	
	<include file="$(find turtlebot2i_bringup)/launch/turtlebot2i_basic.launch" />

	<!--  ************* Moveit config stuff *************  -->

	<!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
	<include file="$(find turtlebot2i_moveit_config)/launch/planning_context.launch">
		<arg name="load_robot_description" value="true"/>
	</include>

	<!-- Run the main MoveIt executable to provide move groups -->
	<include file="$(find turtlebot2i_moveit_config)/launch/move_group.launch">
		<arg name="allow_trajectory_execution" value="true"/>  
		<arg name="fake_execution" value="false"/>
		<arg name="info" value="true"/>
	</include>

	<node name="named_pose_director" pkg="turtlebot2i_arm_director" type="named_pose_director" output="screen">
		<rosparam param="startup_pose">pose_parked</rosparam>
		<rosparam param="floor_pose">[0.078, 0.0, -0.012]</rosparam>
	</node>

</launch>
